backend/app/notify/prune_html.go

```diff
@@ -0,0 +1,122 @@
+package notify
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/net/html"
+)
+
+const commentTextLengthLimit = 100
```

Comment:
This is the only consonant to move to telegram.go, otherwise LGTM.

Reply:
you're right, my bad. moved back to telegram.go